src/shared/lang/ru_RU.json

```
@@ -139,6 +140,7 @@
     "errors": {
       "required": "Заполните поле",
       "email": "Укажите email",
+      "phone": "Укажите телефон в формате +7(232)232-23-23",
```

❗️
Если пишем рыбу, то вот так, а не рандомными цифрами)
```suggestion
      "phone": "Укажите телефон в формате +7(XXX)XXX-XX-XX",
```